server/src/main/java/org/apache/kafka/server/config/ReplicationConfigs.java

```diff
@@ -136,6 +136,10 @@ public class ReplicationConfigs {
     public static final String REPLICA_SELECTOR_CLASS_CONFIG = "replica.selector.class";
     public static final String REPLICA_SELECTOR_CLASS_DOC = "The fully qualified class name that implements ReplicaSelector. This is used by the broker to find the preferred read replica. By default, we use an implementation that returns the leader.";
 
+    public static final String FOLLOWER_FETCH_LAST_TIERED_OFFSET_ENABLE_CONFIG = "follower.fetch.last.tiered.offset.enable";
+    public static final String FOLLOWER_FETCH_LAST_TIERED_OFFSET_ENABLE_DOC = "If enabled, an empty follower will skip replicating offsets up to the last tiered offset and start from the next offset.";
```

Comment:
The doc is not clear, can we update it to make it clear for the users. From KIP:

```
Whether the last tiered offset should be used as the start offset for bootstrapping an empty follower
```

Reply:
I wanted to update the KIP as well - The last tiered offset is actually not the start offset for the follower. It is the next offset.